Frontend/ui-customer/src/Screens/CustomerUtils/MyRidesScreen/Controller.purs

```diff
@@ -200,10 +200,12 @@ myRideListTransformer state listRes = filter (\item -> (item.status == "COMPLETE
     fares = getFares ride.fareBreakup 
     (RideAPIEntity rideDetails) = (fromMaybe dummyRideAPIEntity (ride.rideList !!0))
     baseDistanceVal = (getKmMeter (fromMaybe 0.0 (rideDetails.chargeableRideDistance)))
+    timeVal = (convertUTCtoISC (fromMaybe ride.createdAt ride.rideStartTime) "h:mm A")
+    nightChargesVal = (withinTimeRange "10:00 PM" "5:00 AM" timeVal)
```

Comment:
you can send start time, end time and timeVal as HH:mm:ss (24 hrs) format.

Reply:
Done